### PR TITLE
Require a new Client Cookie when the Client IP changes

### PR DESCRIPTION
**Pull request by wtoorop**

### CODE REVIEW

File: draft-ietf-dnsop-server-cookies.md [hunk 1]
```diff
@@ -164,13 +164,17 @@ The previous example in Appendix A.2 of [@!RFC7873] is NOT RECOMMENDED.
...
 # Constructing a Client Cookie {#clientCookie}
 
-The Client Cookie is a nonce and should be treated as such.  For simplicity, it
-can be calculated from Server IP Address, and a secret known only to the
-Client. The Client Cookie SHOULD have at least 64-bits of entropy.  If a secure
-pseudorandom function (like [@!SipHash-2.4]) is used, there's no need to change
-Client secret often. It is reasonable to change the Client secret only if it has
-been compromised or after a relatively long period of time such as no longer
-than a year.
+The Client Cookie is a **cryptographic** nonce and should be treated as such.
```
Comment (oerdnj) [inline draft-ietf-dnsop-server-cookies.md]: The ** were used only to emphasize the change in email, it doesn’t have to go to RFC.

File: draft-ietf-dnsop-server-cookies.md [hunk 2]
```diff
@@ -185,8 +189,18 @@ Client IP Address as an input to the MAC_Algorithm.  However, when implementing
...
 known.  Therefore, the requirement to put Client IP address as input to was
-removed, and it simply RECOMMENDED to disable the DNS Cookies when privacy is
-required.
+removed.
+
+However, for privacy reasons, in order to prevent tracking of devices across
+links and to not circumvent IPv6 Privacy Extensions [RFC4941], Clients MUST
+NOT re-use a Client or Server Cookie after the Client IP address has changed.
+
+The Client IP address is available on the UDP socket when it received the
```
Comment (oerdnj) [inline draft-ietf-dnsop-server-cookies.md]: receives?

**Review by oerdnj — APPROVED** [review]
Rest looks good to me.

File: draft-ietf-dnsop-server-cookies.md (hunk 1 above)
Comment (wtoorop) [reply draft-ietf-dnsop-server-cookies.md]: Haha, but it's markdown, so will be translated in bold.  I'll remove it though...

**Pull request merged**